### PR TITLE
Add EastBoys Scraper

### PR DESCRIPTION
**Pull request by gamib190**
* Add EastBoys Scraper

* Update Scrapers List

### CODE REVIEW

File: scrapers/EastBoys.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        selector: //div[@class="update_date text-light"]/text()
+        postProcess:
+          - replace:
+              - regex: '(Added:\s)(\d\d\.\d\d\.\d{4})'
+                with: $2
+              - regex: '(\d\d)(\.)(\d\d)(\.)(\d{4})'
+                with: "$1 $3 $5"
+              - regex: '(\d\d)\s(\d\d)\s(\d{4})'
+                with: "$2 $1, $3"
+          - parseDate: 01 02, 2006
```
Comment (JaseNZC) [inline scrapers/EastBoys.yml:22]: Seems a lot of changing, this seems to work ??
```Date:
        selector: //div[@class="update_date text-light"]/text()
        postProcess:
         - parseDate: 02.01.2006

File: scrapers/EastBoys.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      Details:
+        selector: //p[@class="description pr-0 pr-lg-3 text-light"]/text()
+#      Performers:
+#        Name:
+#          selector: //html/body/div[2]/div/div[1]/div/span[2]/b/a
+#      URL: //link[@rel="canonical"][1]/@href
```
Comment (JaseNZC) [inline scrapers/EastBoys.yml:28]: This seems to work for performers as well as adds in tags.
```Performers:
        Name:
          selector: //span[@class="content"]//a
          postProcess:
          - replace:
              - regex: ','
                with: ""
      Tags:
        Name: //div[@class="categories_detail text-light"]/following-sibling::a

File: scrapers/EastBoys.yml (hunk 1 above)
Comment (gamib190) [reply scrapers/EastBoys.yml]: That does seem to work perfectly, so I went ahead and fixed that. This is my first time making a scraper (or anything similar), so thanks again!

File: scrapers/EastBoys.yml (hunk 2 above)
Comment (gamib190) [reply scrapers/EastBoys.yml]: That does seem to work as well, I went ahead and added/fixed that.

File: scrapers/EastBoys.yml (hunk 1 above)
Comment (JaseNZC) [reply scrapers/EastBoys.yml]: No worries I am still learning everyday myself. Thanks for making a contribution as it helps everybody out.

File: scrapers/EastBoys.yml (hunk 2 above)
Comment (JaseNZC) [reply scrapers/EastBoys.yml]: :-)

**Pull request merged**